v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/CassandraAllDataTypesIT.java

```diff
@@ -325,7 +325,7 @@ private Map<String, String> getAllDataTypeMaxRow() {
             "[-99999999999999999999999999999.999999999,0,99999999999999999999999999999.999999999]")
         .put(
             "date_double_map_col",
-            "{\"0\":\"NaN\",\"304\":\"3.14\",\"365\":\"Infinity\",\"365243\":\"0.0\",\"7305\":\"-Infinity\"}")
+            "{\"1970-01-01\":\"NaN\",\"1970-11-01\":\"3.14\",\"1971-01-01\":\"Infinity\",\"1990-01-01\":\"-Infinity\",\"2970-01-01\":\"0.0\"}")
```

Comment:
This is a rvert of the original change right? We switched dates to ints IIRC

Reply:
We still map dates to an integer in Avro. When we represent it as JSON for a map with Cassandra Annotations, it's stringified to the way adapter expects.